tensorflow/contrib/learn/python/learn/io/graph_io_test.py

```diff
@@ -118,8 +118,8 @@ def test_batch_record_features(self):
           _VALID_FILE_PATTERN, batch_size, features, randomize_input=False,
           queue_capacity=queue_capacity, reader_num_threads=2,
           parser_num_threads=2, name=name)
-      self.assertEquals("%s/parse_example_batch_join:0" % name,
-                        features["feature"].name)
+      self.assertEqual("%s/parse_example_batch_join:0" % name,
+                       features["feature"].name)
```

Comment:
nit: you may have already fixed this indentation to two indents for line continuations (like the other statemtnts even in this file)